search/result_processor.py

```diff
@@ -12,7 +12,7 @@
 from .utils import _load_class
 
 DESIRED_EXCERPT_LENGTH = 100
-ELLIPSIS = "&hellip;"
+ELLIPSIS = '<span class="search-results-ellipsis"><i class="icon fa fa-ellipsis-h fa-1"></i></span>'
```

Comment:
The reason that we want to remove ellipses is so that the style of the item can change what it looks like. Having the icon within the content here it the same as adding the ellipsis within the markup.

I would leave a blank span here, and add a `::before` or `::after` style definition that uses the ellipsis font